website/docs/kubernetes-versions.md

```diff
@@ -0,0 +1,20 @@
+---
+id: kubernetes-versions
+title: Kubernetes Versions
+---
+
+Gatekeeper is a policy engine for the [Kubernetes Project](https://kubernetes.io/). To see which version of Kubernetes is compatible with a particular Gatekeeper release, reference the table below.
+
+| Gatekeeper Version | Kubernetes Version                                   |
+|--------------------|------------------------------------------------------|
+| `v3.17`            | `v1.28`,`v1.29`, `v1.30`, `v1.31`                    |
```

Comment:
can we automate creation of this table?

Reply:
example: https://github.com/open-policy-agent/gatekeeper/pull/3537

Reply:
Not sure if @KayenM wants to take this on as a first time contributor. how about we get this PR merged first, then create a separate issue to track? and if @KayenM or others want to volunteer for it, they can work on the follow up?

Reply:
Hey thank you both! Sorry for the delay here.

That's a great point @sozercan, I'd be happy to add that feature as part of this change or after this is merged. It might take me a bit longer to complete if that's alright (end of November)?

Reply:
I am fine with merging this, creating issue for automation. If we are not implementing automation right now, I suggest highlighting this in release guidance - https://github.com/open-policy-agent/gatekeeper/blob/master/docs/RELEASE.md -  untill the automation is in. 

Updating RELEASE.md should be one of the todo in issue tracking the automation.

Reply:
@KayenM bump ^^